frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/vault-standalone.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  serviceAccountJSON,
+  roleBindingJSON,
+  getPVCJSON,
+  configMapJSON,
+  deploymentJson,
+  serviceJSON,
+  routeJSON,
+  networkPolicyJSON,
+  testDeploymentJSON,
+} from '../helpers/vault';
+import { commandPoll } from '../views/common';
+
+export const configureVault = () => {
+  cy.exec('oc get project hashicorp', {
+    failOnNonZeroExit: false,
+  }).then(({ code }) => {
+    // Deploy vault only if doesn't already exist
+    if (code !== 0) {
```

Comment:
```suggestion
```
Is there a reason why we can't assume vault doesn't exist?
Are multiple tests using this?

Reply:
Right now only my test cases are using, in the future, we may add KMS cluster-wide encryption also. Also, while running test cases locally it helps a lot. It saves time.